Makefile

```diff
@@ -122,10 +122,18 @@ proto: ratelimit/gubernator/proto/google ratelimit/gubernator/gubernator.proto $
 	PATH=$$PATH:$(BIN_DIR):$(FIRST_GOPATH)/bin scripts/generate_proto.sh
 
 .PHONY: container-test
+container-test: # Use 'shortcut' to build test image if on Linux, otherwise full build.
+ifeq ($(OS)_$(ARCH), linux_x86_64)
```

Comment:
Hmm this doesn't seem right to me. If I do my development on an ARM machine then this change means I also don't benefit from binaries built as past of the dev flow? Maybe this should check for Linux in general?

Reply:
I adjusted it, although if we want to realistically support running tests on ARM, we would need to have other images we are using available for ARM platform, which does not seem to be the case.

Reply:
Thanks :) yes I think it's a big shame that Thanos and Observatorium images are not multi-arch by default. This is definitely not a best practice and is quite antiquated at this point.

Reply:
I actually thought Thanos can be built for ARM :thinking:, but I see it does not push images for general use. I also think many 'side-services' such as Gubernator are also not readily available for ARM. 

I agree with the sentiment though, would be nice to have the support :+1:.